container-build/cerc-fixturenet-eth-geth/genesis/accounts/accounts.csv

```diff
@@ -0,0 +1,6 @@
+m/44'/60'/0'/0/0,0xe6CE22afe802CAf5fF7d3845cec8c736ecc8d61F,0x02cfc16a6212d7bb9dff89b603afbd856157c367de3ef6b626f205d9b150630ea0,0x888814df89c4358d7ddb3fa4b0213e7331239a80e1f013eaa7b2deca2a41a218
```

Comment:
Should this be in ./config?

Reply:
I think it belongs here, because the accounts are used by `ethereum-genesis-generator` when it generates the genesis file with pre-funded accounts.

Reply:
Can you point me to where that is run?

Reply:
https://github.com/cerc-io/stack-orchestrator/pull/24/files#diff-b2251121f1cc991ed1e0fd066bf19bec47a989e5ac03ad2a189583b72bf8eec8R10

I got my comments backwards on el-config.yaml and accounts.csv.  The latter is a bit more complicated.  Strictly speaking it out to be an *output* of the build step rather than an input, as the keys are predictably generated from the mnemonic in el-config.yaml.

There is an issue here: https://github.com/bozemanpass/merge-testnet/issues/1

Reply:
Here: https://github.com/cerc-io/stack-orchestrator/pull/24/files#diff-87b2242d72cd90189e0e62f38fd4f98deca481208ab2ec476eba14de87e00fe5R25